Source/JavaScriptCore/runtime/BuiltinExecutableMetadata.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+namespace JSC {
+
+class BuiltinExecutableMetadata {
+public:
+    BuiltinExecutableMetadata(const unsigned char* data, unsigned length);
+    BuiltinExecutableMetadata() = default;
```

Comment:
this constructor `BuiltinExecutableMetadata()` should not exist

Reply:
it most definitely should exist because that is how we run it in a generator script